### PR TITLE
Feature/documentation release

### LINKED ISSUE #2372: Docs discoverability clarification
@JJGothard I had a note from Ryan (cust success) re this:

> Unless you use a custom domain, people will not be able to search for or find your documentation online. By default your documentation will only be discoverable to people you explicitly share the URL with, or people browsing the API Network / Postman Templates (if you chose that option during publication).

In https://learning.postman.com/docs/postman/api-documentation/publishing-your-docs/

I can't remember the history but it seems it isn't actually accurate, i.e. you can find docs for example via google search. Think we could either remove it or reframe to just something indicating adding to api network makes your docs more discoverable..?

### PR DESCRIPTION
**Pull request by JJGothard**
Hi @SueSmith , this PR covers the new response headers, authorization, and key-value pair displays in the docs UI that will be released on Friday. Additionally, I took the opportunity of being in these docs to address GitHub issues [#2234](https://github.com/postmanlabs/postman-docs/issues/2344) and #2372. Thanks for your feedback :) (fyi, Raja has been out sick since last week, and I don't anticipate his review. I will still ping him, however, just in case.)

Fixes #2234 
Fixes #2372

### CODE REVIEW

**Comment by SueSmith** [comment]
Thanks @JJGothard if you edit your description on the pr to include "Fixes #2234" and "Fixes #2372" those issues will automatically close when the pr is merged 👍

File: src/pages/docs/postman/api-documentation/authoring-your-documentation.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ You can include information in your API documentation using [Markdown](https://g
 * [Describing collections](#describing-collections)
 * [Documenting request detail](#documenting-request-detail)
     * [Parameters and headers](#parameters-and-headers)
+    * [Documenting required authorization type](#documenting-required-authorization-type)
```
Comment (SueSmith) [inline src/pages/docs/postman/api-documentation/authoring-your-documentation.md:38]: I think I'd just make this "Authorization details" or something shorter like that, the other header at the same level doesn't have "documenting" in it so let's keep them consistent..

File: src/pages/docs/postman/api-documentation/authoring-your-documentation.md [hunk 2]
```diff
@@ -101,10 +102,22 @@ You can add a description to the parameters and headers in your requests—for p
 
 ![Parameter Description](https://assets.postman.com/postman-docs/param-desc.jpg)
 
- The description will appear in your docs, in the request __Params__ and __Headers__ sections, next to the parameter or header name.
+> All key-value pairs will be displayed in your documentation regardless of whether or not they're checked. You can indicate which pairs are required and which are optional in the desription. Consumers of your documentation will be able to choose which key-value pairs they want to try, and they will be able to generate a URL and code snippets based on their selection.
```
Comment (SueSmith) [inline src/pages/docs/postman/api-documentation/authoring-your-documentation.md:105]: * typo "desription" - maybe run through a spell checker before opening prs, I do them all the time too and they're hard to see when you've just written them.. 🙃 
* finding the final sentence here a bit confusing, could we have more detail on what "want to try" means? not entirely clear on how the url and code snippets relate either 🤔

File: src/pages/docs/postman/api-documentation/authoring-your-documentation.md [hunk 3]
```diff
@@ -101,10 +102,22 @@ You can add a description to the parameters and headers in your requests—for p
...
+
+The description will appear in your docs, in the request __Params__ and __Headers__ sections, next to the parameter or header name.
 
 <img alt="Header Description in Docs" src="https://assets.postman.com/postman-docs/header-desc-docs.jpg" width="500px">
 
+### Documenting required authorization type
+
+You can include the type of authorization required to access your APIs in your documentation. You can add your authorization details at the collection level or at the request level. Authorization requirements added at the collection level are persistive, and will be displayed on each request.
```
Comment (SueSmith) [inline src/pages/docs/postman/api-documentation/authoring-your-documentation.md:113]: I'd reword to remove "persistive" maybe "...at the collection level apply to each request" or something

File: src/pages/docs/postman/api-documentation/authoring-your-documentation.md [hunk 4]
```diff
@@ -101,10 +102,22 @@ You can add a description to the parameters and headers in your requests—for p
...
+
+![Docs Link](https://assets.postman.com/postman-docs/Persistive+auth.jpg)
+
+If one of your endpoints requires an authorization that varies from the collection, you can add that authorization to the request in the request tab, and it will be rendered in the documentation.
```
Comment (SueSmith) [inline src/pages/docs/postman/api-documentation/authoring-your-documentation.md:117]: "authorization" > "authorization type"

File: src/pages/docs/postman/api-documentation/authoring-your-documentation.md [hunk 5]
```diff
@@ -113,15 +126,19 @@ Any body data included in your request will appear in the endpoint section of yo
...
+You can also create an example to use for a request. Click **Examples** > **Add Example**.
 
 ![Add Example](https://assets.postman.com/postman-docs/add-example.jpg)
 
-Any data you include in the example will appear in the docs example code sidebar, including body and response data.
+Create the example you want to add and click **Save Example**.
+
+![Body Example](https://assets.postman.com/postman-docs/Create+an+example.jpg)
+
+Any data you include in the example will appear on the right, in the docs example code sidebar, including body and response data.
```
Comment (SueSmith) [inline src/pages/docs/postman/api-documentation/authoring-your-documentation.md:137]: "including body and response data" > "as the response body" or something, the body is the response data

File: src/pages/docs/postman/api-documentation/authoring-your-documentation.md [hunk 6]
```diff
@@ -113,15 +126,19 @@ Any body data included in your request will appear in the endpoint section of yo
...
 
-![Body Example](https://assets.postman.com/postman-docs/body-example.jpg)
+> Learn more about [viewing documentation](https://learning.postman.com/docs/postman/api-documentation/viewing-documentation/).
```
Comment (SueSmith) [inline src/pages/docs/postman/api-documentation/authoring-your-documentation.md:139]: if we can shift to relative link here

File: src/pages/docs/postman/api-documentation/viewing-documentation.md [hunk 7]
```diff
@@ -62,9 +62,9 @@ At the top, you will see any environment shared with the documented collection,
 
 ![View Docs](https://assets.postman.com/postman-docs/SimpleKart+header+expanded+layout+menu.jpg)
 
-You will also see detail on each request, including sample code in various client languages, and an indicator of the method, URL, description, headers, request and response structures, and examples. Documentation authors can optionally include other details.
+You will also see detail on each request, including sample code in various client languages, required authorization type, an indicator of the method, the URL, description, headers, request and response structures and headers, and examples. Documentation authors can optionally include other details.
```
Comment (SueSmith) [inline src/pages/docs/postman/api-documentation/viewing-documentation.md:65]: the first sentence reads like "headers" is repeated, I know it's to indicate they're in the response too but I think I'd just remove "and headers" because the first mention doesn't specify it's request rather than response if that makes sense

File: src/pages/docs/postman/api-documentation/viewing-documentation.md [hunk 8]
```diff
@@ -96,9 +96,9 @@ When you view public documentation, you will see any shared environments, the la
 
 ![View Docs](https://assets.postman.com/postman-docs/SimpleKart+header+expanded+layout+menu.jpg)
 
-For each request, you will see a description of the request, the method and URL, and any headers or parameters. On the right, you will see examples of requests and responses in the language selected.
+For each request, you will see a description of the request, the method and URL, the required authorization type, and any headers or parameters. On the right, you will see request examples and examples of response bodies and headers in the language selected.
```
Comment (SueSmith) [inline src/pages/docs/postman/api-documentation/viewing-documentation.md:99]: I'd reword the second sentence here e.g. "On the right, you will see example client code snippets for the request, together with example response bodies and headers in the language selected."

File: src/pages/docs/postman/api-documentation/authoring-your-documentation.md (hunk 1 above)
Comment (JJGothard) [reply src/pages/docs/postman/api-documentation/authoring-your-documentation.md:38]: Done.

File: src/pages/docs/postman/api-documentation/authoring-your-documentation.md (hunk 2 above)
Comment (JJGothard) [reply src/pages/docs/postman/api-documentation/authoring-your-documentation.md:105]: Good advice on the spell checker -- will def do that 👍. Done and done on both counts here.

File: src/pages/docs/postman/api-documentation/authoring-your-documentation.md (hunk 3 above)
Comment (JJGothard) [reply src/pages/docs/postman/api-documentation/authoring-your-documentation.md:113]: Done.

File: src/pages/docs/postman/api-documentation/authoring-your-documentation.md (hunk 4 above)
Comment (JJGothard) [reply src/pages/docs/postman/api-documentation/authoring-your-documentation.md:117]: Done.

File: src/pages/docs/postman/api-documentation/authoring-your-documentation.md (hunk 5 above)
Comment (JJGothard) [reply src/pages/docs/postman/api-documentation/authoring-your-documentation.md:137]: Done. Also added headers here to be sure to point out that sample response headers will also be there.

File: src/pages/docs/postman/api-documentation/authoring-your-documentation.md (hunk 6 above)
Comment (JJGothard) [reply src/pages/docs/postman/api-documentation/authoring-your-documentation.md:139]: Done, sorry. Got away from me!

File: src/pages/docs/postman/api-documentation/viewing-documentation.md (hunk 7 above)
Comment (JJGothard) [reply src/pages/docs/postman/api-documentation/viewing-documentation.md:65]: Done.

File: src/pages/docs/postman/api-documentation/viewing-documentation.md (hunk 8 above)
Comment (JJGothard) [reply src/pages/docs/postman/api-documentation/viewing-documentation.md:99]: Done

**Comment by JJGothard** [comment]
@SueSmith , all feedback has been incorporated here. Thanks!

**Comment by SueSmith** [comment]
@JJGothard the first "fixes" line in the description is not actually referencing the correct issue number (should be #2344) so if you can close that issue manually - and let's be more careful with this stuff, don't want to close issues accidentally 🙃 

(If you edit the description to see the markdown, it looks like you had included it correctly in the link the first time you mentioned it but not the link text which made it look like it was correct, then that incorrect number was copied to the "fixes" part - nb you don't need to manually add a markdown link to gh issues in the same repo, just the hash character then number will work.)
